src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -39,7 +40,8 @@ export default class MxGraphRenderer {
     try {
       this.insertShapes(bpmnModel.pools);
       this.insertShapes(bpmnModel.lanes);
-      this.insertShapes(bpmnModel.flowNodes);
+      this.insertShapes(bpmnModel.flowNodes.filter(shape => shape.bpmnElement?.kind !== ShapeBpmnElementKind.EVENT_BOUNDARY));
```

Comment:
:question:  why not using `ShapeUtil.isBoundaryEvent` for consistency (otherwise remove this function from `ShapeUtil`)